src/components/TransactionsTab/helpers/clearLocalStorage.ts

```diff
@@ -0,0 +1,31 @@
+import { SubmittedTransaction } from "../../../features/transactions/transactionsSlice";
```

Comment:
There is a hook called `useHistoricalTransactions`. The localStorage transactions should update every time the transactions store is updated.

If for some reason the hook doesn't pick up removed transactions then try to fix the hook.